<launch>
    <!-- Run the plane detector -->
    <node pkg="srs_env_model_percp" type="but_plane_detector" name="but_plane_detector">
        <rosparam file="$(find srs_env_model_percp)/config/planedet_params.yaml" command="load" />
        <remap from="points_in" to="/cam3d/depth/points" />
        <remap from="depth_image_in" to="/cam3d/depth/image_raw" />
        <remap from="camera_info_in" to="/cam3d/rgb/camera_info" />
    </node>
</launch>
